.bcr/presubmit.yml

```diff
@@ -0,0 +1,24 @@
+matrix:
+  platform: ["debian10", "macos", "ubuntu2004"]
+
+tasks:
+  verify_targets:
+    name: "Verify build targets"
+    platform: ${{ platform }}
+    build_targets:
+    - '@protobuf//:all'
+    - '@protobuf//java/core/...'
+    - '@protobuf//java/lite/...'
+    - '-@protobuf//:common_dist_files'
+    - '-@protobuf//:proto_api'
```

Comment:
We should probably run some tests too

Reply:
I'm not sure that any tests will pass on the BCR, and this PR doesn't include that testing. I think we should stick to what was last published in https://github.com/bazelbuild/bazel-central-registry/blob/main/modules/protobuf/23.1/presubmit.yml first, just to un-fork the situation and get the protobuf repo at HEAD to represent the state that's currently published to BCR, then edit after.